README.rst

```diff
@@ -88,11 +86,11 @@ Contributing
 
 |Open Source Helpers|
 
-If you would like to get involved, start by joining the `SunPy mailing list`_ and check out the `Developers Guide`_ section of the SunPy docs.
+If you would like to get involved, start by joining the `SunPy mailing list`_ and check out the `Developers Guide`_ section of the ``sunpy`` docs.
 Stop by our chat room `#sunpy:openastronomy.org`_ if you have any questions.
```

Comment:
Given that the Element channel is far more active than the mailing list, perhaps the Element channel should be listed first?

Also, the "Developer's Guide" is singular possessive in the docs.  Inconsistently, "Newcomers' Guide" is plural possessive...

Reply:
Which one should each one be? We can rename them to be correct.

Reply:
Changed to `Developers' Guide` in the readme, will update the docs later on.